### DISCUSSION TITLE
Show assigned roles

### DISCUSSION THREAD

**Discussion by TechAspirant** [serenity-is/Serenity#5813] [Help] [answered]
Hello,

I want to show assigned roles to a user in user list (User Grid).
Say Eg. I have 3 roles
Admin
Operator
Engineer

And I have 2 Users
User 1
User 2

Out of three roles I assigned two roles (Operator and Engineer ) to User 2
then grid should look like

ID | User Name | Roles
1  | User 1         | No Role assigned
2  | User 2         | Operator, Engineer

**Comment by minhhungit** [comment, marked as answer]
Check this example: https://serenity.is/docs/tutorials/movies/allowing_multiple_genre_selection
Keyword GenreList, GenreListFormatter
